test/common/common.go

```diff
@@ -0,0 +1,230 @@
+/*
```

Comment:
common.go is overly generic.
can we break it out to 
verify.go - to cover all the assert/verify helpers
setup.go - to cover all the setup/creation helpers?

Reply:
I will create a file called setup.go, and put it under the folder `client`.
The file verify.go can be put under the folder resources, since it verifies everything about the knative resources.